setup.py

```diff
@@ -2,9 +2,9 @@
 
 setup(
     name='tidepods',
-    version='0.1',
+    version='1.0',
```

Comment:
... I wouldn't call it `1.0` until it's tried and tested extensively, but you're the maintainer :wink:

Reply:
Also, I think we use MAJOR.MINOR.PATCH (3 places) now everywhere, don't we? https://semver.org/

Major versions are for breaking changes. So if you call something 1.0, you indicate that you are confident that your API (the major function signatures and choice of abstractions) is well thought through and will be stable for a while - which you only know after extensive trial, like @dionhaefner said. :-)

Reply:
Noted, updated to MAJOR.MINOR.PATCH and back to 0.1.0 for now